client_util.py

```diff
@@ -92,9 +94,19 @@ def monitoring(address,info=None):
         logger.warn('error getting state', exc_info=True)
 
 def config_options_dict(config):
-    config_dict = {}
+    """
+    Parsing config file
+
+    Args:
+        config: Pythong config parser object
+
+    Returns:
+        A dict with the different sections of the config file
+        and the literal values of the configuraton objects
+    """
+    config_dict = dict()
```

Comment:
Also note this article:
https://doughellmann.com/blog/2012/11/12/the-performance-impact-of-using-dict-instead-of-in-cpython-2-7-2/